app/controllers/hostgroups_controller.rb

```diff
@@ -5,6 +5,7 @@ class HostgroupsController < ApplicationController
   before_filter :find_resource,  :only => [:nest, :clone, :edit, :update, :destroy]
   before_filter :ajax_request,   :only => [:process_hostgroup, :puppetclass_parameters]
   before_filter :taxonomy_scope, :only => [:new, :edit, :process_hostgroup]
+  before_filter :taxonomy_warnings, :only => [:edit]
```

Comment:
Only run this filter if `Taxonomy.enabled_taxonomies.present?`, which ensures there's at least one taxonomy enabled